GTG/gtk/ui/preferences.ui

```diff
@@ -0,0 +1,497 @@
+                            <property name="can_focus">False</property>
+                            <property name="valign">center</property>
+                            <property name="receives_default">False</property>
+                            <signal name="font-set" handler="on_fontbutton_font_set" swapped="no"/>
+                          </object>
+                          <packing>
+                            <property name="expand">False</property>
+                            <property name="fill">False</property>
+                            <property name="pack_type">start</property>
+                            <property name="position">1</property>
+                          </packing>
+                        </child>
+                        <child>
+                          <object class="GtkBox">
+                            <property name="orientation">vertical</property>
+                            <property name="valign">center</property>
+                            <property name="vexpand">false</property>
+                            <property name="visible">true</property>
+                            <property name="margin-start">10</property>
+                            <child>
+                              <object class="GtkLabel" id="font_title">
+                                <property name="label">Set font type and size</property>
+                                <property name="visible">true</property>
+                                <property name="xalign">0.0</property>
+                                <attributes>
+                                  <attribute name="weight" value="bold"/>
+                                </attributes>
+                              </object>
+                            </child>
+                            <child>
+                              <object class="GtkLabel" id="font_label">
+                                <property name="label">Set the font type and size to be used in task editor</property>
+                                <property name="visible">true</property>
+                                <property name="xalign">0.0</property>
+                                <style>
+                                  <class name="dim-label"/>
+                                </style>
+                                <attributes>
+                                  <attribute name="scale" value="0.833333"/>
+                                </attributes>
+                              </object>
+                            </child>
+                          </object>
+                        </child>
+                      </object>
+                    </child>
+                  </object>
+                </child>
+              </object>
+              <packing>
+                <property name="name">general</property>
+                <property name="title" translatable="yes">General Settings</property>
```

Comment:
I think so, might save some troubles in the future in different languages.
Added the property